README.md

```diff
@@ -1,5 +1,17 @@
 # AWS Ansible-based docker development environment for Firefox Accounts
 
+Table of Contents (ToC)
+======================
```

Comment:
Let's not have 2 `h1` elements in the same doc and omit this `Table of Contents` heading entirely. It should be obvious without that what these links are.